packages/gestalt/src/RadioGroup.js

```diff
@@ -0,0 +1,64 @@
+// @flow strict
+import { type Node } from 'react';
+import Fieldset from './Fieldset.js';
+import Flex from './Flex.js';
+import RadioGroupButton from './RadioGroupButton.js';
+import { RadioGroupContextProvider } from './RadioGroupContext.js';
+
+type Props = {|
+  /**
+   * The individual RadioGroup.RadioButtons for this group
```

Comment:
We should probably mention that children can be grouped if necessary. Also worth mentioning any other components that would be valid in here, like Label or any others (and specifically call out which components should _not_ by valid). Something like

> A collection of RadioGroup.RadioButton representing the available options, as well as any Labels or ____ if needed. Other components such as ____ should _not_ be included. Note that children can be grouped into organizational components if desired.